### DISCUSSION TITLE
blocking and large dataset

### DISCUSSION THREAD

**Discussion by theimanph** [moj-analytical-services/splink#2001] [General]
Hi, 

I have a very large dataset several hundreds of millions of rows. Does it make sense to subsample the data (take a million rows), learn a model and then apply it to the whole dataset? Or is there a better way? I tried to use blocking on the whole dataset 2 days later, it was still not finished and I got kicked off.. Any ideas would be greatly appreciated!!

Sincerely,

tom

**Comment by Kameron-Eck** [comment]
I have the same situation. Would love to hear some feedback.
